### PR TITLE
bytes: add support for base64 encoded flags

### PR DESCRIPTION
**Pull request by glerchundi**
Signed-off-by: Gorka Lerchundi Osa <glertxundi@gmail.com>

### CODE REVIEW

File: bytes.go [hunk 1]
```diff
@@ -103,3 +104,100 @@ func BytesHex(name string, value []byte, usage string) *[]byte {
 func BytesHexP(name, shorthand string, value []byte, usage string) *[]byte {
 	return CommandLine.BytesHexP(name, shorthand, value, usage)
 }
+
+// BytesBase64 adapts []byte for use as a flag. Value of flag is Base64 encoded
+type bytesBase64Value []byte
+
+func (bytesBase64 bytesBase64Value) String() string {
```
Comment (eparis) [inline bytes.go]: Can I get godoc style comments on everything exported? Then I'll merge.

Comment (glerchundi) [reply bytes.go]: Done.

**Comment by glerchundi** [comment]
@eparis PTAL. I added only a comment referencing which interface is implementing, Is that OK for you?

**Comment by glerchundi** [comment]
Thanks @eparis!

**Pull request merged**